src/types/providerTypes.ts

```diff
@@ -189,7 +192,9 @@ export type PandaProviderType = {
   getSocialProfile: () => Promise<SocialProfile | undefined>;
   getBalance: () => Promise<Balance | undefined>;
   getOrdinals: () => Promise<Ordinal[] | undefined>;
-  sendBsv: (params: SendBsv[]) => Promise<string | undefined>;
+  sendBsv: (
+    params: SendBsv[]
+  ) => Promise<{ rawtx: string; txid: string } | undefined>;
```

Comment:
To follow existing pattern, let's pull these into their own type `SendBsvResponse`